src/walltime.cc

```diff
@@ -147,7 +167,8 @@ WallTimeImp::WallTimeImp()
       cycles_per_second_(0), seconds_per_cycle_(0.0),
       last_adjust_time_(0), drift_adjust_(0),
       max_interval_cycles_(0) {
-  cycles_per_second_ = static_cast<int64_t>(CyclesPerSecond());
+    const double kMaxErrorInterval = 100e-6;
```

Comment:
if you push to the branch from which this PR was created, the code should update.